spec/concurrent/set_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'set' 
+module Concurrent
+  describe Set do 
+    let!(:set) { described_class.new }
+
+    it 'concurrency' do
+      (1..THREADS).map do |i|
+        Thread.new do
+            set << i
```

Comment:
Change this whole spec to look like `Hash` spec: https://github.com/ruby-concurrency/concurrent-ruby/blob/df482db36caf1b0c1d69a8ff97a2407469e1e315/spec/concurrent/hash_spec.rb#L8 (by adding another inner `1000.times do |j|` loop, just like for `Hash`).